tools/install_golangci_lint_deps.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# just a warning - as "sudo" is not in the container we are using
+echo "This script should be run as root"
+
+apt-get update
+
+# This is needed to lint internal/upload/koji package
+apt-get install -y libkrb5-dev
```

Comment:
(quick drive-by comment) doing this in one long command line (like `apt install -y libkrb5-dev libgpgme-dev libbtrfs-dev libdevmapper-dev` is slightly more efficient as it will read/rebuild the apt cache only once (but of course writing comments is a little bit harden then.

Reply:
yes, I just took it over (was like this) due to the comments - I also didn't really like calling apt that often